platform/containers/kubernetes-distro/Dockerfile

```diff
@@ -5,8 +5,9 @@ ENV DOCKER_VERSION 1.12.6
 ENV DOCKER_SHA256 cadc6025c841e034506703a06cf54204e51d0cadfae4bae62628ac648d82efdd
 # Current version.
 ENV KUBERNETES_VERSION=1.6.7
+ENV KUBERNETES_SERVER_HASH=29828b8913fc0d922f42cb7b04d15a518984e0d6
 # All supported versions, including backward compatible versions.
-ENV KUBERNETES_VERSIONS_LIST="1.5.7 1.6.7"
+ENV KUBERNETES_VERSIONS_LIST="1.6.7"
```

Comment:
Can we get server binary URL as input, which will leave space for customized kubernetes build.

(If this'd been the case, software_info package related things needs to be changed as well)

Reply:
Yes. That is possible. But I'd like to discourage customized kubernetes build.